src/server/game/World/World.cpp

```diff
@@ -3210,8 +3210,7 @@ void World::LoadGlobalPlayerDataStore()
     QueryResult result = CharacterDatabase.Query("SELECT guid, account, name, gender, race, class, level FROM characters WHERE deleteDate IS NULL");
     if (!result)
     {
-        sLog->outString();
```

Comment:
I guess so :D

Reply:
Then why remove it? have you seen the output in the console yet ?

Reply:
Ok so normally it looks like this

```
2017-12-12 23:17:21 Load Global Player Data...
2017-12-12 23:17:21
2017-12-12 23:17:21 ERROR: >>  Loaded 0 Players data!
```

now it will look like this
```
2017-12-12 23:17:21 Load Global Player Data...
2017-12-12 23:17:21 >>  Loaded 0 Players data.
```
So it's okay.